mkdocs.yml

```diff
@@ -33,6 +33,7 @@ extra_css:
 
 plugins:
   - search
+  - mermaid2
```

Comment:
We need to install this plugin `mkdocs-mermaid2-plugin` in our docs dependencies to get the docs build working again:

https://github.com/nsidc/earthaccess/blob/91866ac18e806e2d53d6b68cd000fa372c3e010f/pyproject.toml#L90-L123